mu-plugins/blocks/latest-news/src/edit.js

```diff
@@ -21,14 +21,20 @@ import ServerSideRender from '@wordpress/server-side-render';
  * @return {WPElement} Element to render.
  */
 export default function Edit( { attributes, setAttributes, name } ) {
-	const { perPage } = attributes;
+	const { blogId, perPage } = attributes;
 
 	const onPerPageChange = ( value ) => setAttributes( { perPage: value * 1 } );
+	const onBlogIdChange = ( value ) => setAttributes( { blogId: Number( value ) } );
 
 	return (
 		<div { ...useBlockProps() }>
 			<InspectorControls>
 				<PanelBody title={ __( 'Settings', 'wporg' ) }>
+					<NumberControl
+						label={ __( 'Blog Id', 'wporg' ) }
+						onChange={ onBlogIdChange }
+						value={ blogId }
```

Comment:
Maybe add these for some quick references?
```suggestion
						value={ blogId }
						help={ __(
							'For example, 8 for wordpress.org/news, 719 for developer.wordpress.org/news',
							'wporg'
						) }
```